js/web/lib/wasm/jsep/webgpu/ops/fuse-utils.ts

```diff
@@ -15,7 +15,8 @@ export const getActivationSnippet = (attributes: InternalActivationAttributes, i
 } => {
   switch (attributes.activation) {
     case 'Relu':
-      return {activationFunction: '', applyActivation: 'value = max(value, 0.0);'};
+      return {activationFunction: '',
+        applyActivation: isVec4 ? 'value = max(value, vec4(0.0));' : 'value = max(value, 0.0);'};
```

Comment:
Do we need to make corresponding changes to Sigmodi also?

Reply:
will do a seperate PR for Sigmod, need to cook up some test data (for relu I have models that run into it)